src/diffusers/utils/testing_utils.py

```diff
@@ -1147,3 +1179,13 @@ def update_mapping_from_spec(device_fn_dict: Dict[str, Callable], attribute_name
         update_mapping_from_spec(BACKEND_EMPTY_CACHE, "EMPTY_CACHE_FN")
         update_mapping_from_spec(BACKEND_DEVICE_COUNT, "DEVICE_COUNT_FN")
         update_mapping_from_spec(BACKEND_SUPPORTS_TRAINING, "SUPPORTS_TRAINING")
+        update_mapping_from_spec(BACKEND_RESET_PEAK_MEMORY_STATS, "RESET_PEAK_MEM_STATS")
+
+
+@require_torch
+def flush_memory(device: str, gc_collect=False, reset_mem_stats=False):
+    if gc_collect:
+        gc.collect()
+    if reset_mem_stats:
+        backend_reset_peak_memory(device)
+    backend_empty_cache(device)
```

Comment:
```suggestion
```
Let's keep `gc.collect()` in `setUp`/`tearDown` and import `backend_empty_cache` etc.